vllm/utils.py

```diff
@@ -1108,6 +1112,104 @@ def parse_args(self, args=None, namespace=None):
 
         return super().parse_args(processed_args, namespace)
 
+    @staticmethod
+    def _pull_args_from_config(args: List[str]) -> List[str]:
+        """
+            method to pull arguments specified in the config file
+            into the commandline args variable.
+            
+            The arguments in config file will be inserted between 
+            the argument list.
+            
+            example:
+            ```yaml
+                port: 12323
+                tensor-parallel-size: 4
+            ```
+            ```python
+            $: vllm {serve,chat,complete} "facebook/opt-12B" \
+                --config config.yaml -tp 2
+            $: args = [
+                "serve,chat,complete",
+                "facebook/opt-12B", 
+                '--config', 'config.yaml', 
+                '-tp', '2'
+            ]
+            $: args = [
+                "serve,chat,complete",
+                "facebook/opt-12B", 
+                '--port', '12323', 
+                '--tensor-parallel-size', '4', 
+                '-tp', '2'
+                ]
+            ```
+
+            Please note how the config args are inserted after the sub command.
+            this way the order of priorities is maintained when these are args 
+            parsed by super().
```

Comment:
There seems to be extra indentation here. The content should have the same indentation as the triple quotes.

Reply:
changed and noted. same pattern with `_load_confug_file` as well.